client/state/analytics/actions.js

```diff
@@ -50,12 +57,25 @@ export const recordEvent = ( service, args ) => ( {
 	}
 } );
 
+export const recordEventOnce = ( eventId, service, args ) => ( dispatch, getState ) => {
+	const state = getState();
```

Comment:
I thought about using a module variable, the code is a lot simpler, but I thought that this adds another global "state", and the idea of Redux is having a single global state. I'm ok using `state`from the middleware. 

To be honest, I don't have a precise idea of the best way to achieve this.